package.json

```diff
@@ -37,9 +37,85 @@
     "prettier": "^3.0.3",
     "rimraf": "^5.0.1",
     "typescript": "^5.2.2",
-    "typescript-eslint": "^7.8.0"
+    "typescript-eslint": "^7.8.0",
+    "wireit": "^0.14.9"
   },
   "dependencies": {
     "parse5": "^7.0.0"
+  },
+  "exports": {
+    ".": {
+      "import": "./lib/main.js",
+      "require": "./commonjs/main.js"
```

Comment:
To make typescript happy, I think we need two of everything including the types:

- the root package is interpreted as esm
- so `*.js` is assumed to be esm, and `*.dts` the types for esm
- `*.cjs` and `*.cts` would be cjs

I think [attw](https://arethetypeswrong.github.io/) will be unhappy otherwise 😬

I could be wrong though